go/service/kbfs_favorites_handler.go

```diff
@@ -0,0 +1,85 @@
+// Handlers for ephemeral-related gregor messages
+
+package service
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/gregor"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/gregor1"
+)
+
+const kbfsFavoritesHandlerName = "kbfsFavoritesHandler"
+
+type kbfsFavoritesHandler struct {
+	libkb.Contextified
+
+	// handled stores the set of requests that this client has processed so
+	// that we don't handle a request more than once.
+	handled map[string]bool
```

Comment:
I don't know anything about gregor handlers, so maybe this is a standard practice, but it seems like this map can grow without bound.  Memory leak?

Also, it's not protected by a lock, but maybe that's fine if these handlers are only ever invoked by a single goroutine.

Reply:
You definitely shouldn't do this.

Instead, locally dismiss the message.  I'll put a comment where that goes.